save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -182,10 +182,16 @@ class DownloadFilesController(
     @ApiResponse(responseCode = "404", description = "File is not found.")
     @PostMapping(path = ["/internal/files/download-save-agent"], produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
```

Comment:
maybe add a GetMappint to the same path? just for testing in browser as you suggested

Reply:
Addressed with 9aef36aa403eca710ac80fb661d42aee0ddbcc06